app/services/DisplayPropertiesService.scala

```diff
@@ -81,16 +83,37 @@ class DisplayPropertiesService @Inject() (val graphqlConfiguration: GraphQLConfi
 
     val propertyType: String = attributes.find(_.attribute == "PropertyType").getStringValue
 
-    DisplayProperty(active, componentType, dataType, description, displayName, editable, group, guidance, label, multiValue, ordinal, p.propertyName, propertyType, unitType)
+    DisplayProperty(
+      active,
+      componentType,
+      dataType,
+      description,
+      displayName,
+      editable,
+      group,
+      guidance,
+      label,
+      multiValue,
+      ordinal,
+      p.propertyName,
+      propertyType,
+      unitType,
+      summary
+    )
   }
 
-  private def displayPropertyFilter(dp: DisplayProperty, metadataType: String): Boolean = {
-    dp.active && dp.propertyType.toLowerCase == metadataType.toLowerCase
+  private def displayPropertyFilter(dp: DisplayProperty, metadataType: Option[String]): Boolean = {
+    dp.active && metadataType.forall(`type` => dp.propertyType.equalsIgnoreCase(`type`))
   }
 
-  def getDisplayProperties(consignmentId: UUID, token: BearerAccessToken, metadataType: String): Future[List[DisplayProperty]] = {
+  def getDisplayProperties(consignmentId: UUID, token: BearerAccessToken, metadataType: Option[String]): Future[List[DisplayProperty]] = {
```

Comment:
As mentioned earlier could move the caching to here, if its possible.

Reply:
We can do this definitely but I feel it isn't required at the moment.